### PR TITLE
Improve install/usage

### PR DESCRIPTION
**Pull request by MisterTea**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -104,13 +108,7 @@ pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch
 pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html
 ```
 
-After that, you will need to generate the python code of the thrift config definition. If you changed the thrift later on, you will have to rerun this.
```
Comment (kittipatv) [inline docs/installation.md]: We will still need this section. Thrift compiler can be installed with anaconda:

`conda install thrift-cpp`

File: docs/installation.md [hunk 2]
```diff
@@ -58,41 +58,45 @@ Now you can run the tests:
...
+Install Horizon dependencies:
 ```
 pip install -r requirements.txt
 ```
 
-Then, install appropriate PyTorch 1.0 nightly build into the virtual environment:
+Then, install appropriate PyTorch 1.0 nightly build:
 ```
 # For CPU build
 pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```
Comment (kittipatv) [inline docs/installation.md]: Maybe update this section to use conda instead?

**Comment by MisterTea** [comment]
Addresses #56

File: docs/installation.md [hunk 3]
```diff
@@ -1,123 +1,120 @@
...
-docker build -t horizon:dev .
+git clone https://github.com/facebookresearch/Horizon.git
+cd Horizon/
 ```
-On Linux you can build the image with specific memory allocations from command line:
+
+Install dependencies:
 ```
-docker build -t horizon:dev --memory=8g --memory-swap=8g .
+conda install `cat requirements.txt`
```
Comment (kittipatv) [inline docs/installation.md]: `conda install --file requirements.txt`

File: ml/rl/test/test_tensorboardX.py [hunk 4]
```diff
@@ -25,7 +25,7 @@ def test_writing(self):
             with summary_writer_context(writer):
                 SummaryWriterContext.add_scalar("test", torch.ones(1))
             writer.add_scalar.assert_called_once_with(
-                "test", torch.ones(1), global_step=0
+                "test", torch.ones(1), global_step=SummaryWriterContext._global_step
```
Comment (kittipatv) [inline ml/rl/test/test_tensorboardX.py]: I'd prefer to keep this explicit.

File: requirements.txt [hunk 5]
```diff
@@ -1,19 +1,13 @@
...
 protobuf==3.6.1
-pyglet==1.3.2
 pytest-xdist==1.24.0
-python-dateutil==2.7.3
-pytz==2018.5
-requests==2.19.1
+pytorch-nightly
 scipy==1.1.0
-six==1.11.0
+tensorboard==1.9.0
```
Comment (kittipatv) [inline requirements.txt]: I tried installing tensorboard (from conda). It won't run without tensorflow.

File: docs/usage.md [hunk 6]
```diff
@@ -1,12 +1,25 @@
 # Usage
 
-## 1 - Online RL Training
-Horizon supports online training environments for model testing. To train a model on OpenAI Gym, simply run:
+Horizon is designed for large-scale, distributed recommendation/optimization tasks where we don't have access to a simulator.  In this environment, it's typically better to train offline on batches of data, and release new policies slowly over time.  Because the policy updates slowly and in batches, we use *off-policy* algorithms.  To test a new policy without deploying it, we rely on *counter-factual policy evaluation (CPE)*, a set of techniques for estimating a policy based on the actions of another policy.
```
Comment (kittipatv) [inline docs/usage.md]: Nice addition. I wonder if you could add newlines to make it text-editor friendly (keep it <100 chars)?

File: docker/cpu/Dockerfile [hunk 7]
```diff
@@ -21,23 +20,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
-    libprotobuf-dev \
-    libsnappy-dev \
-    locales \
-    openmpi-bin \
-    openmpi-doc \
-    protobuf-compiler \
     sudo \
     software-properties-common \
     vim \
+    emacs \
```
Comment (kittipatv) [inline docker/cpu/Dockerfile]: 👍🏽

**Pull request closed without merging**